paimon-core/src/main/java/org/apache/paimon/operation/metrics/ScanMetrics.java

```diff
@@ -66,10 +67,25 @@ private void registerGenericScanMetrics() {
         metricGroup.gauge(
                 LAST_SCAN_RESULTED_TABLE_FILES,
                 () -> latestScan == null ? 0L : latestScan.getResultedTableFiles());
+        metricGroup.gauge(
+                MANIFEST_HIT_CACHE,
+                () -> cacheMetrics == null ? 0L : cacheMetrics.getHitObject().get());
+        metricGroup.gauge(
+                MANIFEST_MISSED_CACHE,
+                () -> cacheMetrics == null ? 0L : cacheMetrics.getMissedObject().get());
     }
 
     public void reportScan(ScanStats scanStats) {
         latestScan = scanStats;
         durationHistogram.update(scanStats.getDuration());
     }
+
+    public void reportCache(CacheMetrics cacheMetrics) {
+        this.cacheMetrics = cacheMetrics;
```

Comment:
Why set cacheMetrics again? You should use this cacheMetrics...

Reply:
Just want to make the code looks unified....